scripts/type_check.sh

```diff
@@ -2,12 +2,4 @@
 
 set -eu
 
-HERE="$(dirname "$(readlink -f "$BASH_SOURCE")")"
-REPO_ROOT="$(realpath "${HERE}/..")"
-
-cd "${REPO_ROOT}"
-
-source "${REPO_ROOT}/scripts/library/venv.sh"
-use_venv "type_check" frozen_type_check_requirements.txt
-
-python3 -m mypy .
+poetry run mypy .
```

Comment:
Does poetry always run from the root directory?

Reply:
No